cohttp-async/test/test_async_integration.ml

```diff
@@ -40,7 +41,18 @@ let server =
          Pipe.close w;
          r
        in
-       Server.respond_with_pipe ~code:`OK body
+       Server.respond_with_pipe ~code:`OK body >>| response
+    )
+  ]
+  @ [ (* pipelined_expert *)
```

Comment:
I think these `x @ y @ z` constructions are more efficient as `x :: y :: z` where possible, since you are constructing a single-element list and then using `@` (which isn't tail recursive)

Reply:
Updated.